solc/CommandLineParser.cpp

```diff
@@ -582,7 +583,12 @@ General Information)").c_str(),
 	outputFormatting.add_options()
 		(
 			g_strPrettyJson.c_str(),
-			"Output JSON in pretty format. Currently it only works with the combined JSON output."
+			"Output JSON in pretty format."
+		)
+		(
+			g_strPrettyJsonIndent.c_str(),
+			po::value<uint32_t>()->value_name("n")->default_value(solidity::util::JsonPrintingFormat::defaultIndent),
+			"JSON pretty print indent."
```

Comment:
I'd add a bit more detail:
```suggestion
			po::value<uint32_t>()->value_name("N")->default_value(util::JsonPrintingFormat::defaultIndent),
			"Indent pretty-printed JSON with N spaces. Only valid in combination with --pretty-json."
```

Also, you do not have to include the `solidity::` namespace since almost everything is inside and it's just noise.